### ISSUE TITLE
Add time tags for dates and times

### ISSUE THREAD

**Issue by Nicd** [Nicd/code-stats#11]
Dates and times should have `<time>` tags so they have semantic meaning for screen readers for example.

**Comment by Nicd** [comment]
Fixed in 1.4.0.
